src/components/scroll-indicator.js

```diff
@@ -5,7 +5,6 @@ import { scroll } from './_shared/animation';
 import { flexCenter } from './_shared/styled-mixins';
 
 const StyledContainer = styled.div`
-  position: absolute;
```

Comment:
dont remove the absolute, its supposed to sit on the bottom of the page initial pageView i.e. bottom of hero section which is full viewport height